src/components/Form.js

```diff
@@ -184,17 +208,19 @@ class Form extends React.Component {
                 >
                     <View style={[this.props.style]}>
                         {this.childrenWrapperWithProps(this.props.children)}
+                        {this.props.isSubmitButtonVisible && (
                         <FormAlertWithSubmitButton
                             buttonText={this.props.submitButtonText}
                             isAlertVisible={_.size(this.state.errors) > 0 || Boolean(this.props.formState.error)}
                             isLoading={this.props.formState.isLoading}
-                            message={this.props.formState.error}
+                            message={_.isEmpty(this.props.formState.errorFields) ? this.getErrorMessage() : null}
```

Comment:
I figured I should add this check here to avoid displaying a duplicate message on both the error field above the submit button and below the errored field. Instead, we will display the `fixTheErrors` message and the field error message below the input.